.lgtm.yml

```diff
@@ -0,0 +1,8 @@
+extraction:
+  python:
+    python_setup:
+      version: "3"
```

Comment:
LGTM needed to be told to use python 3 to import the code base. Without this their import step fails.